vpr/src/place/place_constraints.cpp

```diff
@@ -11,6 +11,9 @@
 #include "globals.h"
 #include "place_constraints.h"
 
+/// @brief sentinel value for indicating that a subtile has not been specified
+constexpr int NO_SUBTILE = -1;
```

Comment:
put this in a header file that both region.cpp and place_constraints.cpp both include. Logically in region.h